### DISCUSSION TITLE
Configuring MelSpectrogram parameters for custom dataset

### DISCUSSION THREAD

**Discussion by astricks** [coqui-ai/TTS#400] [General Q&A]
I've been trying to figure out the a good configuration for training the Tacotron2 model. I'm not sure how to set MelSpectrogram parameters accurately.

Specifically, how would I calculate the right values for mel_fmin and mel_fmax for my dataset?

Thanks!

**Comment by erogol** [comment]
just go with the def values, or you can  use https://github.com/coqui-ai/TTS/blob/main/notebooks/dataset_analysis/CheckSpectrograms.ipynb to find good params for your dataset. Check spectrograms and see how well it looks (freq level separated and fewer background noise) and listen resynthesized audio with different parameters. 

(Pls don't ask questions in issues. Post here or use our chat room)

**Reply by astricks** [reply]
Aah ok, thank you! I’ll use the discussions for questions going forward.
